JS-test.js

```diff
@@ -0,0 +1,48 @@
+// 1 Programming Basics
+
+let x
+for (x = 10; x < 40; x++) {
+  if (x % 5 == 0 && x % 3 == 0) {
+    console.log(x + ' Jackpot!')
+  } else if (x % 3 == 0) {
+    console.log(x + ' This is divisible by 3')
+  } else if (x % 5 == 0) {
+    console.log(x + ' This is divisible by 5')
+  } else {
+    console.log(x)
+  }
+}
+
+// 2 DOM manipulation
+let body = document.querySelector('body')
+let button = document.createElement('button')
+button.appendChild(document.createTextNode('Button'))
+button.onclick = function () {
+  const h1 = createElement('h1')
+  h1.innerHTML = 'This is an h1 tag'
+  body.appendChild(h1)
+}
```

Comment:
Very good, though not complete. The way you added the button and created a click handling function is great! But you just needed to also add into the function that the button should now be disabled, and its color should be changed. Also, you did not append the button to the body, thus it will never be displayed.